provider/datagen/src/transform/uprops/mod.rs

```diff
@@ -21,14 +21,20 @@
 
 mod bin_uniset;
 #[cfg(feature = "experimental")]
+mod canonical_decompositions;
+#[cfg(feature = "experimental")]
 mod casemapping;
+#[cfg(feature = "experimental")]
+mod decompositions_serde;
 mod enum_codepointtrie;
 mod script;
 mod uprops_helpers;
-mod uprops_serde;
+pub(crate) mod uprops_serde;
```

Comment:
Thought: if `uprops_serde` is common between `uprops` and `collator`, it should be moved up one level. You can call it `icu_toml`. 

That said, will collator data actually come from a different data source than properties data, or will it also come from `icu_exportdata`? In that case we wouldn't need separate `SourceData` fields, and it would also make sense to have all of this in one module (call it `icu_export` or something).

Reply:
The normalization data will come from the `icuexportdata` tool in ICU4C. At least for now, the collation data will come from the `genrb` tool in ICU4C. (The necessary code is a much better fit for new flags in `genrb` instead of duplicative code in `icuexportdata`.) So: Different executable but still executables that are both part of the ICU4C repo.

Reply:
Reverted to non-pub, because other changes made this change unnecessary.